### PR TITLE
(SIMP-4009) Add support for Puppet 5 and OEL

### PR DESCRIPTION
**Pull request by jeefberkey**
SIMP-4009 #comment update simp_rsyslog

### CODE REVIEW

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-simp_rsyslog/pipelines/28653741

**Comment by ayohrling** [comment]
Getting iptables failures on OEL. Looking into it.

**Comment by ayohrling** [comment]
Alright, so just an update; the system for OEL has firewalld configured out of the box, and allows pings through in ordering prior to logging them, causing there to be no iptables.log file on the system. Upon checking out ways to handle this, the acceptance tests are jamming things into iptables with a 'TODO' to move to the iptables module. I've been trying to get this converted over now. Hopefully should have something tomorrow pushed.

**Comment by ayohrling** [comment]
Marking as work in progress; hopefully will have changes pushed up shortly

**Comment by ayohrling** [comment]
https://gitlab.com/simp/pupmod-simp-simp_rsyslog/pipelines/29090110

File: metadata.json [hunk 1]
```diff
@@ -43,6 +43,13 @@
         "6",
```
Comment (lnemsick-simp) [inline metadata.json]: You forgot to bump puppet version range

Comment (nick-markowski) [reply metadata.json]: Updated

**Pull request merged**